src/deferred.js

```diff
@@ -212,20 +215,26 @@ jQuery.extend( {
 										}
 									};
 
+							processQueue.push( process );
+
 							// Support: Promises/A+ section 2.3.3.3.1
 							// https://promisesaplus.com/#point-57
 							// Re-resolve promises immediately to dodge false rejection from
 							// subsequent errors
 							if ( depth ) {
-								process();
+								deferredTick();
```

Comment:
I *think* this specific line could actually change the execution order of `process`s.

Previously this single `process` would be invoked before other scheduled async ones. Now it will invoke other scheduled ones first, then this one last, but it is still "immediately" (see comment a couple line above).

Can anyone think of a case where this is wrong?